plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -417,6 +419,32 @@ export class MongoDBInstrumentation extends InstrumentationBase<
     span.setAttribute(SemanticAttributes.DB_STATEMENT, JSON.stringify(query));
   }
 
+  /**
+   * Triggers the response hook in case it is defined.
+   * @param span The span to add the results to.
+   * @param config The MongoDB instrumentation config object
+   * @param result The command result
+   */
+  private _handleExecutionResult(
+    span: Span,
+    config: MongoDBInstrumentationConfig,
+    result: CommandResult
+  ) {
+    if (typeof config.responseHook === 'function') {
+      safeExecuteInTheMiddle(
+        () => {
+          config.responseHook!(span, { data: result });
+        },
+        err => {
+          if (err) {
+            diag.error('Mongodb instrumentation: Error running response hook', err);
```

Comment:
you could use `this._diag` instead to avoid having to add manually the `Mongodb instrumentation` prefix

Reply:
Did not find any references in the contrib project for such usage, and I also `this._diag` does not some to exist (TS compile fails).
Could you share some reference please?
Thanks!

Reply:
OK, I see this is new from 0.22 - fixed!
Thanks :)